pkg/handler/loki.go

```diff
@@ -68,20 +68,25 @@ func GetFlows(cfg LokiConfig) func(w http.ResponseWriter, r *http.Request) {
 
 		for key := range params {
 			regexStr := strings.Builder{}
-			for _, value := range strings.Split(params.Get(key), ",") {
-				if strings.Contains(value, dateRangeDelimiter) {
-					var rangeValues = strings.Split(value, dateRangeDelimiter)
-					//add start param if specified
-					if len(rangeValues[0]) > 0 {
-						extraArgs.WriteString(startParam)
-						extraArgs.WriteString(rangeValues[0])
-					}
-					//add end param if specified
-					if len(rangeValues[1]) > 0 {
-						extraArgs.WriteString(endParam)
-						extraArgs.WriteString(rangeValues[1])
-					}
+
+			param := params.Get(key)
+			//add start / end param if specified
+			if key == "startTime" {
```

Comment:
I'd be in favor of setting this `"startTime"`, `"endTime"` etc literals as constants.

I'd also change the `if-else` chain by a `switch`